### PR TITLE
update to osde2e test harness image

### PR DESCRIPTION
**Pull request by MStokluska**
**What**
This PR aims to automate the creation of osde2e integreatly test harness image and increase the timeout required to run our tests to 7000seconds.

### CODE REVIEW

**Comment by MStokluska** [comment]
@mikenairn could you please let me know if you are happy with the changes too?

File: ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml [hunk 1]
```diff
@@ -68,6 +68,8 @@ periodics:
       command:
       - /osde2e
       env:
+      - name: POLLING_TIMEOUT
+        value: "7000"
```
Comment (mikenairn) [inline ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml:72]: This isn't our job, i doubt you need to be changing it

File: core-services/image-mirroring/integr8ly/mapping_integr8ly_operator [hunk 2]
```diff
@@ -1,2 +1,3 @@
 registry.svc.ci.openshift.org/integr8ly/integreatly-operator:integreatly-operator quay.io/integreatly/integreatly-operator:master
-registry.svc.ci.openshift.org/integr8ly/integreatly-operator:integreatly-operator-test-harness quay.io/integreatly/integreatly-operator-test-harness:master
\ No newline at end of file
+registry.svc.ci.openshift.org/integr8ly/integreatly-operator:integreatly-operator-test-harness quay.io/integreatly/integreatly-operator-test-harness:master
+registry.svc.ci.openshift.org/integr8ly/integreatly-operator:integreatly-operator-test-harness-osde2e quay.io/integreatly/managed-api-test-harness:osde2e
```
Comment (mikenairn) [inline core-services/image-mirroring/integr8ly/mapping_integr8ly_operator:3]: Why the change of name 'integreatly-operator-test-harness-osde2e' -> 'managed-api-test-harness'?

File: ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml [hunk 3]
```diff
@@ -22,6 +22,13 @@ images:
       - destination_dir: .
         source_path: /go/src/github.com/integr8ly/integreatly-operator/build
   to: integreatly-operator
+- dockerfile_path: Dockerfile.osde2e
+  from: os
+  inputs:
+    openshift_release_golang-1.13:
+      as:
+      - registry.svc.ci.openshift.org/openshift/release:golang-1.13
+  to: integreatly-operator-test-harness-osde2e
```
Comment (mikenairn) [inline ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml:31]: Looks fine. You will need one of these to approve changes here https://github.com/openshift/release/blob/master/ci-operator/config/integr8ly/integreatly-operator/OWNERS

File: ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml (hunk 1 above)
Comment (MStokluska) [reply ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml:72]: Hi Mike, currently the default polling timeout is set to 3600 seconds, we need around 6000 seconds for our installation + tests to complete. This aims to increase the default timeout.

File: core-services/image-mirroring/integr8ly/mapping_integr8ly_operator (hunk 2 above)
Comment (MStokluska) [reply core-services/image-mirroring/integr8ly/mapping_integr8ly_operator:3]: good point, thanks, the name change isn't really required.

**Review by philbrookes — APPROVED** [review]
